webapp/app.py

```diff
@@ -286,16 +287,33 @@ def standalone_examples_index():
     )
 
 
-@app.route("/docs/examples/standalone/<path:example_path>")
-def standalone_example(example_path):
+@app.route("/docs/examples/<path:example_path>")
+def example(example_path, is_standalone=False, is_raw=False):
     try:
+        # If the user has requested the raw template, serve it directly
+        if is_raw:
+            raw_example_path = f"../templates/docs/examples/{example_path}.html"
+            # separate directory from file name so that flask.send_from_directory() can prevent malicious file access
+            raw_example_directory = os.path.dirname(raw_example_path)
+            raw_example_file_name = os.path.basename(raw_example_path)
+            return flask.send_from_directory(raw_example_directory, raw_example_file_name, mimetype="text/raw", max_age=86400)
+
         return flask.render_template(
-            f"docs/examples/{example_path}.html", is_standalone=True
+            f"docs/examples/{example_path}.html", is_standalone=is_standalone
         )
-    except jinja2.exceptions.TemplateNotFound:
+    except (jinja2.exceptions.TemplateNotFound, werkzeug.exceptions.NotFound):
         return flask.abort(404)
 
 
+@app.route("/docs/examples/standalone/<path:example_path>")
+def standalone_example(example_path):
+    return example(example_path, is_standalone=True)
+
+
+@app.route("/examples/<path:example_path>")
```

Comment:
It may be good to make it a bit more explicit that this route returns "raw" text of example template. Maybe: `/raw/examples/<path:example_path>` or `/docs/examples/<path:example_path>?raw=true`

For some time I was considering moving the examples pages from `/docs/examples` to `/examples` (because they seem a bit unnecesarily nested under docs), so reserving such path for raw examples would prevent us from having such option in future (or at least make the migration harder).

Reply:
I've updated this to use a `raw=true` query parameter